JustTweak/Classes/Extensions/TweakExtensions.swift

```diff
@@ -5,6 +5,21 @@
 
 import Foundation
 
+extension String {
```

Comment:
Maybe we could reduce the code in textDidChange() if we combined the functionality and then assign the value using `value = type(of: text.underlyingValue)
`
```
extension String {

    var underlyingValue: Any {
        if let int = Int(self) {
            return int
        } else if let double = Double(self) {
            return double
        } else if let float = Float(self) {
            return float
        } else {
            return self
        }
    }
}
```

Reply:
<img width="1075" alt="Screenshot 2021-04-28 at 14 04 16" src="https://user-images.githubusercontent.com/11852767/116408380-9feaf380-a82a-11eb-9431-a6ae4d19f1ce.png">

Reply:
Also im not sure the Float condition will ever get executed when initialising a Double first with a string representation of both Double/Float